measurements/api/measurements.py

```diff
@@ -27,9 +27,10 @@
 
 from measurements import __version__
 from measurements.config import REPORT_INDEX_OFFSET, REQID_HDR, request_id
-from measurements.models import Report, Input, Measurement, Autoclaved
+from measurements.models import Report, Input, Measurement, Autoclaved, Fastpath
 
 MSM_ID_PREFIX = 'temp-id'
+FASTPATH_MSM_ID_PREFIX = 'tid-'
```

Comment:
Can we make this prefix more clearly temporary and related to the fastpath?

Maybe `temp-id-fast-`?

Reply:
I was trying to keep it short to keep the URL easier to handle and copypaste. temp-id-fast will match "temp-id" startswith() and might break something. `temp-fid` ?

Reply:
> temp-id-fast will match "temp-id" startswith() and might break something. 
That's a good point.

`temp-fid` looks reasonable to me.

I mostly suggest to keep `temp` in the prefix so that it's clear that this ID is a temporary one and will likely change in the future.

Reply:
> temp-id-fast will match "temp-id" startswith() and might break something.

That's a good point.

`temp-fid` looks reasonable to me.

I mostly suggest to keep temp in the prefix so that it's clear that this ID is a temporary one and will likely change in the future.

Reply:
Done